consensus/polybft/polybft.go

```diff
@@ -435,6 +435,21 @@ func (p *Polybft) Initialize() error {
 	return nil
 }
 
+func ForkManagerInitialParamsFactory(config *chain.Chain) (*chain.ForkParams, error) {
+	pbftConfig, err := GetPolyBFTConfig(config)
+	if err != nil {
+		return nil, err
+	}
+
+	return &chain.ForkParams{
```

Comment:
We have identified a couple of more parameteres when we were elaborating on-chain governance RFC: https://www.notion.so/polygontechnology/RFC-191-f40a27ea359b46f78f67e22c0627b0e6?pvs=4#a57804ec80b6433eaaecaa845b0c9507 (e.g. block-gas-limit/target and checkpoint-interval, which does not exist at the moment, but aim was to introduce it with on chain governance).

Reply:
we csn easily add parameters later when we need them. personally i would leave only one or two in struct for now

Reply:
We agree that we will add all  parameters needed through another pr